gammapy/irf/effective_area.py

```diff
@@ -358,15 +370,18 @@ def __init__(
 
         if interp_kwargs is None:
             interp_kwargs = self.default_interp_kwargs
-        axes = [
-            BinnedDataAxis(
-                energy_lo, energy_hi, interpolation_mode="log", name="energy"
-            ),
-            BinnedDataAxis(
-                offset_lo, offset_hi, interpolation_mode="linear", name="offset"
-            ),
-        ]
-        self.data = NDDataArray(axes=axes, data=data, interp_kwargs=interp_kwargs)
+
+        e_edges = np.append(energy_lo, energy_hi[-1]).value * energy_lo.unit
+        energy_axis = MapAxis.from_edges(e_edges, interp="log", name="energy")
+
+        # TODO: for some reason the H.E.S.S. DL3 files contain the same values for offset_hi and offset_lo
+        if np.allclose(offset_lo.to_value("deg"), offset_hi.to_value("deg")):
```

Comment:
Why is the `if` `else` statement required now?

Reply:
As far as I understood the DL3 files contain the center values for the offset bins. As this is not supported by the current DL3 data format, the same (center) values are written in `offset_hi` and `offset_lo` columns of the FITS file.  That's why I added this check...or did you mean something else?